lib/independence/local_storage_namespace.js

```diff
@@ -6,7 +6,7 @@ class LocalStorageNamespace {
   }
 
   exists (key) {
-    return this._storage[this._fullKey(key)] !== undefined
+    return Boolean(this._storage.getItem(this._fullKey(key)))
```

Comment:
we might still get the private case of an empty string. I think there is no way to get around it and write
`item != undefined` (with 1 equal) will be true for values that aren't `null` or `undefined`

Reply:
I think the very existance of a key in the storage is enough to return `true` to `exists`.

Reply:
The only problem with the `Boolean` version is a false negative when an empty string is stored.
you can try on the browser:
```js
localStorage.foo = ''
Boolean(localStorage.foo)
localStorage.foo != undefined
```

Reply:
I used `null` instead of `undefined` becaue the linter accepts it and it works the same.